src/black/output.py

```diff
@@ -8,34 +8,40 @@
 from mypy_extensions import mypyc_attr
 import tempfile
 
-from click import echo, style
+from click import echo, style as _style
```

Comment:
Is this renamed because of the function parameter shadowing? We could change the parameter name instead.

Reply:
If change the parameter name, it would change how we specify the `OutputLevel` to the `black.output.out` function, and since `click.style` is only used once in the whole file, I think this is fine.